{"name": "Create a membership invalid account id. -- @1.1 ", "status": "failed", "statusDetails": {"message": "compare.UnmetExpectation: Expected 400 to equal 404\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\project_steps.py\", line 41, in step_impl\n    expect(int(status_code)).to_equal(context.response.status_code)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 173, in to_equal\n    ensure(self.value == other, True, message)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 151, in ensure\n    raise UnmetExpectation(message)\n"}, "steps": [{"name": "Given I set up a \"POST\" request to \"/accounts/{string}/memberships\" endpoint", "status": "passed", "start": 1556562161488, "stop": 1556562161489}, {"name": "And I set up the data", "status": "passed", "attachments": [{"name": ".text", "source": "6ea26746-1979-4af4-85c9-98f048febe4f-attachment.txt", "type": "text/plain"}], "start": 1556562161489, "stop": 1556562161491}, {"name": "When I send the request", "status": "passed", "start": 1556562161491, "stop": 1556562161928}, {"name": "Then I get a \"400\" status code as response", "status": "failed", "statusDetails": {"message": "compare.UnmetExpectation: Expected 400 to equal 404\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\project_steps.py\", line 41, in step_impl\n    expect(int(status_code)).to_equal(context.response.status_code)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 173, in to_equal\n    ensure(self.value == other, True, message)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\compare.py\", line 151, in ensure\n    raise UnmetExpectation(message)\n"}, "start": 1556562161929, "stop": 1556562161931}, {"name": "And I should see a messages error: The path you requested has no valid endpoint.", "status": "skipped", "start": 1556562161932, "stop": 1556562161932}], "parameters": [{"name": "id", "value": "string"}], "start": 1556562160216, "stop": 1556562161932, "uuid": "b5dff2e5-7041-4fba-ba7b-bdffd9ee76f6", "historyId": "1ffd687b5488f1353bee54a5ad23195d", "labels": [{"name": "tag", "value": "functional_post_negative"}, {"name": "tag", "value": "get_id_first_account"}, {"name": "tag", "value": "account"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Functional test cases Negative to create account membership"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}